packages/expo-keep-awake/src/index.ts

```diff
@@ -26,11 +38,13 @@ export function useKeepAwake(tag: string = ExpoKeepAwakeTag): void {
  * If the `tag` argument is specified, the screen will not sleep until you call `deactivateKeepAwake`
  * with the same `tag` argument. When using multiple `tags` for activation you'll have to deactivate
  * each one in order to re-enable screen sleep. If tag is unspecified, the default `tag` is used.
+ *
+ * Web support [is limited](https://caniuse.com/wake-lock).
+ *
  * @param tag *Optional* - Tag to lock screen sleep prevention. If not provided, the default tag is used.
  */
-
-export function activateKeepAwake(tag: string = ExpoKeepAwakeTag): void {
-  if (ExpoKeepAwake.activate) ExpoKeepAwake.activate(tag);
+export async function activateKeepAwake(tag: string = ExpoKeepAwakeTag): Promise<void> {
```

Comment:
maybe create new method `activateKeepAwakeAsync` then keep the old one around and add a deprecation warning?